scripts/verify-npm-publish.sh

```diff
@@ -0,0 +1,30 @@
+# default to latest if no dist tag is passed in
+dist_tag=${1:-latest}
+
+package_name=$(node --eval "
+    const PACKAGE = './package.json';
+    let pkg = require(PACKAGE);
+    console.log(pkg.name);
+")
+
+local_version=$(node --eval "
+    const PACKAGE = './package.json';
+    let pkg = require(PACKAGE);
+    console.log(pkg.version);
+")
+
+npm_public_registry_version=$(npm view $package_name version $dist_tag);
+
+echo "package name: $package_name"
+echo "dist tag: $dist_tag"
+echo "local version: $local_version"
+echo "npm version: $npm_public_registry_version\n"
+
+while [ "$local_version" != "$npm_public_registry_version" ]
```

Comment:
Can we time-out after 5 mins or something? Imagining a scenario where we keep pinging forever due to some failure, and start getting rate limited :)

Reply:
Good point! On it now.